game-app/game-core/src/main/java/games/strategy/triplea/odds/calculator/OrderOfLossesInputPanel.java

```diff
@@ -1,5 +1,7 @@
 package games.strategy.triplea.odds.calculator;
 
+import static games.strategy.triplea.util.UnitSeparator.getComparatorUnitCategories;
```

Comment:
style nitpick, avoid static method imports unless in test code and importing things like assertions.  The reasoning is that when inspecting the code, the method call looks like a call to a local method or to a super class.

Reply:
It might be even cleaner to have a static method in  `UnitSeparato` to do the sorting, rather than returning the comparator to then let the client code to the sorting. EG: `UnitSeparator.sort(  <unit-type-list> )`